crates/pera-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -1028,6 +1029,47 @@ impl AuthorityPerEpochStore {
         })
     }
 
+    pub fn get_encrypted_decryption_key_shares(&self) -> PeraResult<Vec<Vec<u8>>> {
+        if self.epoch() == FIRST_EPOCH_ID {
+            return Err(PeraError::Unknown(
+                "First epoch does not have decryption key shares, need to run network DKG"
+                    .to_string(),
+            ));
+        }
+
+        let encrypted_decryption_key_shares = match self.epoch_start_state() {
+            EpochStartSystemState::V1(data) => data.get_encrypted_decryption_key_shares(),
+        };
+        Ok(encrypted_decryption_key_shares.ok_or(PeraError::Unknown(
+            "Decryption key shares not found".to_string(),
+        ))?)
+    }
+
+    pub fn get_decryption_key_share(&self) -> PeraResult<Vec<u8>> {
```

Comment:
doc

Reply:
I will delete this function from this PR as I already changed & fixed. Also it is not needed yet.